server/routes/routes.go

```diff
@@ -25,12 +25,13 @@ func v1Routes(r gin.IRouter) {
 	r.GET("/namespaces/:namespace/isbsvcs", handler.ListInterStepBufferServices)
 	r.GET("/namespaces/:namespace/isbsvcs/:isbsvc", handler.GetInterStepBufferService)
 	r.GET("/namespaces/:namespace/pipelines/:pipeline", handler.GetPipeline)
-	r.GET("/namespaces/:namespace/pipelines/:pipeline/vertices", handler.ListVerices)
+	r.GET("/namespaces/:namespace/pipelines/:pipeline/vertices", handler.ListVertices)
 	r.GET("/namespaces/:namespace/pipelines/:pipeline/vertices/:vertex", handler.GetVertex)
 	r.GET("/namespaces/:namespace/pipelines/:pipeline/vertices/:vertex/pods", handler.ListVertexPods)
 	r.GET("/namespaces/:namespace/pods/:pod/log", handler.PodLogs)
 	r.GET("/metrics/namespaces/:namespace/pods", handler.ListPodsMetrics)
 	r.GET("/metrics/namespaces/:namespace/pods/:pod", handler.GetPodMetrics)
 	r.GET("/namespaces/:namespace/pipelines/:pipeline/edges", handler.ListPipelineEdges)
 	r.GET("/namespaces/:namespace/pipelines/:pipeline/edges/:edge", handler.GetPipelineEdge)
+	r.GET("/namespaces/:namespace/pipelines/:pipeline/vertex/:vertex", handler.GetVertexMetrics)
```

Comment:
Is `/namespaces/:namespace/pipelines/:pipeline/vertices/:vertex/metrics` working?